mrchecker-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/mrchecker/webapi/endpoint/soap/FarenheitToCelsiusMethod_Request_FromCode.java

```diff
@@ -1,110 +0,0 @@
-package com.capgemini.mrchecker.webapi.endpoint.soap;
```

Comment:
Why delete this file and many other below?

Reply:
The deleted files are leftovers of the System Test tests that has been moved to: https://github.com/devonfw/mrchecker example project.